.doks/doks-cluster/Makefile

```diff
@@ -52,10 +52,10 @@ populate_cluster: setup_vars create_terraformrc
 
 .PHONY: remove_codacy
 remove_codacy: set_cluster_context
```

Comment:
this commands should be in the same makefile as the the install

Reply:
this makefile on this directory seems focused on creating a doks cluster (with or without codacy)